src/OpenTelemetry.Extensions.Hosting/OpenTelemetryBuilder.cs

```diff
@@ -61,13 +62,13 @@ internal OpenTelemetryBuilder(IServiceCollection services)
         Guard.ThrowIfNull(configure);
 
         this.Services.ConfigureOpenTelemetryMeterProvider(
-            (sp, builder) => builder.ConfigureResource(configure));
+            builder => builder.ConfigureResource(configure));
```

Comment:
What's this change for?

Reply:
Just some cleanup. Originally we only had a single "Configure" method which required you accept the two parameters. Later on we added a second one which doesn't require you to use the `IServiceProvider`.